src/osdc/Objecter.h

```diff
@@ -1206,10 +1216,12 @@ class Objecter : public md_config_obs_t, public Dispatcher {
 
     bool used_replica = false;
     bool paused = false;
+    bool use_osd_epoch = false;
 
     int osd = -1;      ///< the final target osd, or -1
 
     epoch_t last_force_resend = 0;
+    epoch_t epoch; ///< Required epoch
```

Comment:
how about naming it required_interval (since we care about intervals changing, not a new map that doesn't affect our pg), defaulting to 0, and getting rid of use_osd_epoch?